odoo/addons/test_lint/tests/_odoo_checker_gettext.py

```diff
@@ -43,9 +43,14 @@ class OdooBaseChecker(BaseChecker):
             'gettext-placeholders',
             'Use keyword arguments when you have multiple placeholders',
         ),
+        'E8506': (
+            'Usage of %r in _, _lt function',
+            'gettext-repr',
+            'Don\'t use %r to automatically insert quotes in translation strings',
```

Comment:
I'd add a little explanation, "Quotes can be different depending on the language, they must be part of the translated string."